src/UI/MessageButtons/Button.module.css

```diff
@@ -1,16 +1,16 @@
 .button {
 	display: block; /* Makes sure our buttons are shown from top to bottom */
-	border: none;
-	cursor: pointer; /* Somehow these buttons don't have this as the default */
-	margin: .3em 0;
+	width: 100%;
 	padding: 5px;
-	background: inherit;
-	color: var(--parley-link-color, #4891F5);
+	border: none;
 	border: 1px;
+	border-style: solid;
 	border-color: var(--parley-link-color, #4891F5);
+	margin: 0.3em 0;
+	background: inherit;
 	border-radius: 10px;
-	border-style: solid;
-	width: 100%;
+	color: var(--parley-link-color, #4891F5);
```

Comment:
Should the color not be: #2E2EFF? 
Same as in `src/UI/Message.module.css`
`color: var(--parley-link-color, #2E2EFF);`

Probably a reason why you used this new color.
P.S. if you gonna change this color don't forget border-color!

Reply:
Good find!
Fixed 26b2f3f01e74aefe09d1171fe9798b12cb0ac2be also added missing css vars to cdn index 30b6cb9e0338b9587c85a4c8a0dddde0a3775414